src/ucp/core/ucp_ep.c

```diff
@@ -739,7 +740,7 @@ void ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config)
             if (!ucp_ep_is_tag_offload_enabled(config)) {
                 /* Tag offload is disabled, AM will be used for all
                  * tag-matching protocols */
-                ucp_ep_config_set_rndv_thresh(worker, config, config->key.rndv_lane,
+                ucp_ep_config_set_rndv_thresh(worker, config, config->key.rndv_lanes[0],
```

Comment:
need to add TODO to set the threshold based on all lanes

Reply:
added